adapters/amqp-vertx/src/main/java/org/eclipse/hono/adapter/amqp/Application.java

```diff
@@ -10,7 +10,7 @@
  *
  * SPDX-License-Identifier: EPL-2.0
  *******************************************************************************/
-package org.eclipse.hono.adapter.amqp.impl;
+package org.eclipse.hono.adapter.amqp;
```

Comment:
IMHO this will create the *split package* problem when loading classes of the same package from two different signed jar files ...
For the other adapters that you have migrated already, we kept the spring related classes in the `impl` package, so I suggest that we do that here as well.